client/layout/masterbar/checkout.jsx

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
It depends where we want the logic for determining the cart based on the `cart`/`selectedSite` to live. If we want it in the minimal masterbar component (whatever we name it), I think it should be separate, because otherwise, doing presence checks for `cart`/`selectedSite` seems like a messy way to determine that we're in checkout. Changing the API of this component to accept a `url` prop, moving the logic to determine that URL into (either) layout, seems to solve both issues.

I'll rename this back to `MasterbarMinimal` and update the PR with that change.